pkg/network/netlink/conntracker.go

```diff
@@ -251,6 +251,8 @@ func (ctr *realConntracker) register(c ct.Con) int {
 		ctr.state[key] = formatIPTranslation(transTuple, generation)
 	}
 
+	log.Debugf("src=%s dst=%s sport=%d dport=%d src=%s dst=%s sport=%d dport=%d", c.Origin.Src, c.Origin.Dst, *c.Origin.Proto.SrcPort, *c.Origin.Proto.DstPort, c.Reply.Src, c.Reply.Dst, *c.Reply.Proto.SrcPort, *c.Reply.Proto.DstPort)
```

Comment:
How often do we expect this to log if a customer enables debug logging? Consider changing it to `Tracef` if its high volume

Reply:
It is pretty high volume actually. Let me change it to `Tracef`.

Reply:
Done